src/components/molecules/TrackCard/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Typography, Box } from '@mui/material'
+import { Colors } from 'src/styles/color'
+import Image, { StaticImageData } from 'next/image'
+
+export interface TrackCardProps {
+  id?: string
+  title: string
+  minute?: number
+  speaker?: string
+  speakerIcon?: string | StaticImageData
+  color?: string
+  idColor: string
+  fullWidth: boolean
+}
+
+export const TrackCard = ({
+  id,
+  title,
+  minute,
+  speaker,
+  speakerIcon,
+  color = Colors.background.secondary_pink,
+  idColor = Colors.background.primary_pink,
+  fullWidth = false
+}: TrackCardProps) => {
+  return (
+    <Box position="relative">
+      <Box
+        sx={{
+          backgroundColor: color,
+          // TODO(@maito1201): fullWidthがtrueの時の挙動はタイムテーブルの時刻表示を考慮したものを実装する
+          width: fullWidth ? '100vw' : '100%',
+          padding: '16px',
+          position: 'absolute',
+          borderRadius: '4px'
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            backgroundColor: idColor,
+            borderRadius: '2px',
+            width: 'fit-content',
+            padding: '0 4px'
+          }}
+        >
+          <Typography variant="caption" color={Colors.text.white}>
+            {id}
+          </Typography>
+        </Box>
+        <Box mb="4">
```

Comment:
タイトルとスピーカーアイコンの間の 4px の余白が取れてなさそうです。 `mb` で指定する場合は `mb="4px"` のように単位まで含める必要があります。

---
あるいは、それぞれに margin-bottom を指定するのではなく、親要素を flexbox か grid layout にして gap あるいは row-gap を付与することで、 ID とタイトル、タイトルとアイコンの間の余白を一括で指定できますが、差分が大きくなるので省略します。

```suggestion
        <Box mb="4px">
```




<img width="966" alt="image" src="https://user-images.githubusercontent.com/40013676/232325338-91e1d77b-261d-4cbe-897a-4ce7fe077252.png">